src/docfx/lib/ProcessUtility.cs

```diff
@@ -145,7 +144,7 @@ public static Task<string> Execute(string fileName, string commandLineArgs, stri
             {
                 try
                 {
-                    return new FileStream(lockPath, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.Delete);
+                    return new FileStream(lockPath, FileMode.CreateNew, FileAccess.ReadWrite, FileShare.Delete);
```

Comment:
can [fileoptions.deleteonclose](https://docs.microsoft.com/en-us/dotnet/api/system.io.fileoptions?view=netframework-4.7.1#System_IO_FileOptions_DeleteOnClose) (aka [FILE_FLAG_DELETE_ON_CLOSE](https://docs.microsoft.com/en-us/windows/desktop/api/FileAPI/nf-fileapi-createfilea)) handle process crashes in between the using statement?